### ISSUE TITLE
Apostrophe inside tag interpolation operator

### ISSUE THREAD

**Issue by SN4T14** [jadejs/jade#2182]
If you have something like this:

```jade
p.
    #[a(href="https://example.com") this don't work]
```

Jade spits out this error:

```
Jade:2
    1| p.
  > 2|     #[a(href="https://example.com") this don't work]

The end of the string was reached with no closing bracket found.
```

Which seems to imply that it thinks the apostrophe in "don't" is the start of a string. This is not an issue if you have an even number of apostrophes inside the operator:

```jade
p.
    #[a(href="https://example.com") this don't don't work]
```

Which results in the expected HTML:

```html
<p><a href="https://example.com">this don't don't work</a></p>
```

This issue happens both on jade-lang.com and on a local Jade 1.11.0.

**Comment by alejonext** [comment]
Thats [woks](http://codepen.io/alejonext/pen/ZQGeVJ?editors=100), look the tabs

**Comment by ForbesLindesay** [comment]
This is fixed in the upcoming 2.0.0 release
